cfme/tests/cloud/test_providers.py

```diff
@@ -597,11 +598,38 @@ def test_refresh_with_empty_iot_hub_azure(request, provider, setup_provider):
     assert result.validate(wait="60s")
 
 
+@pytest.fixture
+def azure_available_regions(provider):
+    def _func():
+        view = navigate_to(AzureProvider, "Add")
+        view.browser.refresh()
+        # prefill the provider type to enable regions dropdown
+        view.fill({'prov_type': provider.type.capitalize()})
+        return [opt.text for opt in view.region.all_options][1:]  # first option is '<Choose>'
```

Comment:
As below, I don't think this should be a fixture, for the same reasons.

Possibly a property of the AzureProvider class, but even then I'm not sure of the utility outside of these specific test cases.

I appreciate your approach/effort to minimize code duplication though!